.vscode/tasks.json

```diff
@@ -38,6 +38,58 @@
                 "command": "${config:python.pythonPath}",
             },
             "problemMatcher": []
+        },
+        {
+            "label": "Create change note",
+            "type": "process",
+            "command": "python3",
+            "args": [
+                "misc/scripts/create-change-note.py",
+                "${input:language}",
+                "${input:name}",
+                "${input:category}"
+            ],
+            "presentation": {
+                "reveal": "never",
+                "close": true
+            },
+            "problemMatcher": []
+        }
+    ],
+    "inputs": [
+        {
+            "type": "pickString",
+            "id": "language",
+            "description": "Language",
+            "options":
+            [
+                "go",
+                "java",
+                "javascript",
+                "cpp",
+                "csharp",
+                "python",
+                "ruby",
+                "swift",
+            ]
+        },
+        {
+            "type": "promptString",
+            "id": "name",
+            "description": "Name"
+        },
+        {
+            "type": "pickString",
+            "id": "category",
+            "description": "Category",
+            "options":
+            [
+                "minorAnalysis",
+                "newQuery",
+                "fix",
+                "majorAnalysis",
+                "breaking",
```

Comment:
Note that the categories differ between library changes and query changes: https://github.com/github/codeql/blob/main/docs/change-notes.md

Reply:
(continued from a reply to your other comment)... given that the categories supported for `lib` and `src` change notes differ, the list of categories would have to be a union of the two sets of categories, and this would be a confusing mess.

Also, it doesn't appear that there's any way to do branching inside tasks, so there would be no way to prompt the user to select `lib` or `src` and show the appropriate list of categories for that change.

So here's what I'll do instead: I'll create _two_ tasks, one for query changes and one for library changes, and then each task can have its own list of categories (and ensure that the new change note is created in the correct place).

I consider changes notes for shared libraries to be out of scope for this PR. If someone wants to add such support later on, great! (In that case, I would suggest adding it as a third, separate task.)

Reply:
> So here's what I'll do instead: I'll create two tasks, one for query changes and one for library changes, and then each task can have its own list of categories (and ensure that the new change note is created in the correct place).

Sounds good.

> I consider changes notes for shared libraries to be out of scope for this PR. If someone wants to add such support later on, great! (In that case, I would suggest adding it as a third, separate task.)

Makes sense.